tests/integration/go_ethereum/test_goethereum_ipc.py

```diff
@@ -25,11 +22,8 @@
 
 
 def _geth_command_arguments(geth_ipc_path, base_geth_command_arguments):
-    geth_port = get_open_port()
     yield from base_geth_command_arguments
     yield from (
-        "--port",
-        geth_port,
```

Comment:
we can probably remove `--rpc.enabledeprecatedpersonal` from here

Reply:
Also tried without and no luck.